host: MacBook-Pro-von-Frederic.local
Building DAG of jobs...
Using shell: /opt/local/bin/bash
Provided cores: 8
Rules claiming more threads will be scaled down.
Job stats:
job          count
---------  -------
cythonize        1
er_cx            1
er_exp           1
er_pyr           1
total            4

Select jobs to execute...
Execute 1 jobs...

[Fri Nov  8 16:24:39 2024]
localrule cythonize:
    input: src/py_raccoon/balance_sampling.pyx, src/py_raccoon/balance_spanning_trees.pyx
    output: results/__cythonized__
    jobid: 2
    reason: Missing output files: results/__cythonized__
    resources: tmpdir=/var/folders/dj/14c985vd1ng7gf0vvw_ytrpw0000gn/T

[Fri Nov  8 16:24:41 2024]
Finished job 2.
1 of 4 steps (25%) done
Select jobs to execute...
Execute 2 jobs...

[Fri Nov  8 16:24:41 2024]
localrule er_pyr:
    input: results/__cythonized__
    output: results/synthetic/er/nodes=10 p=0.5 r=0.5/pyr/samples=100 specified_p=0.5/pyr_0.csv
    jobid: 1
    reason: Missing output files: results/synthetic/er/nodes=10 p=0.5 r=0.5/pyr/samples=100 specified_p=0.5/pyr_0.csv; Input files updated by another job: results/__cythonized__
    wildcards: kind=er, n_nodes=10, prob_p=0.5, neg_edge_prob=0.5, n_samples=100, pyr_spec_edge_prob=0.5, run=0
    resources: tmpdir=/var/folders/dj/14c985vd1ng7gf0vvw_ytrpw0000gn/T

[Fri Nov  8 16:24:41 2024]
localrule er_cx:
    input: results/__cythonized__
    output: results/synthetic/er/nodes=10 p=0.5 r=0.5/cx/samples=10 exact=False directed=False parallel=False/cx_1.csv
    jobid: 3
    reason: Missing output files: results/synthetic/er/nodes=10 p=0.5 r=0.5/cx/samples=10 exact=False directed=False parallel=False/cx_1.csv; Input files updated by another job: results/__cythonized__
    wildcards: kind=er, n_nodes=10, prob_p=0.5, neg_edge_prob=0.5, n_samples=10, exact=False, directed=False, parallel=False, run=1
    resources: tmpdir=/var/folders/dj/14c985vd1ng7gf0vvw_ytrpw0000gn/T

RuleException:
CalledProcessError in file /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/workflow/Snakefile, line 37:
Command 'set -euo pipefail;  /Users/fredericbusch/miniforge3/envs/py-raccoon/bin/python3.12 /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/.snakemake/scripts/tmphmy29bse.experiment.py' returned non-zero exit status 1.
RuleException:
CalledProcessError in file /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/workflow/Snakefile, line 47:
Command 'set -euo pipefail;  /Users/fredericbusch/miniforge3/envs/py-raccoon/bin/python3.12 /Users/fredericbusch/Desktop/Thesis/forks/py-raccoon/.snakemake/scripts/tmp1uzu817b.experiment.py' returned non-zero exit status 1.
[Fri Nov  8 16:24:47 2024]
[Fri Nov  8 16:24:47 2024]
Error in rule er_pyr:
    jobid: 1
    input: results/__cythonized__
    output: results/synthetic/er/nodes=10 p=0.5 r=0.5/pyr/samples=100 specified_p=0.5/pyr_0.csv

Error in rule er_cx:
    jobid: 3
    input: results/__cythonized__
    output: results/synthetic/er/nodes=10 p=0.5 r=0.5/cx/samples=10 exact=False directed=False parallel=False/cx_1.csv

Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2024-11-08T162439.835526.snakemake.log
WorkflowError:
At least one job did not complete successfully.
